photon-api/src/integTest/scala/com/linkedin/photon/ml/estimators/GameEstimatorIntegTest.scala

```diff
@@ -384,7 +386,28 @@ class GameEstimatorIntegTest extends SparkTestUtils with GameTestUtils {
    *
    * @return A loaded data frame
    */
-  def getData(spark: SparkSession): DataFrame = spark.read.parquet(inputPath)
+  def getData(spark: SparkSession): DataFrame = {
+    val df = spark.read.parquet(inputPath)
+
+    // Convert any mllib.Vector instances to ml.Vector
+    val rows = df.rdd.map { row =>
+      val vals = row.toSeq.map {
+        case vector: SparkSparseVector => vector.asML
+        case other => other
+      }
+
+      Row.fromSeq(vals)
+    }
+
+    val schema = new StructType(df.schema.map { field =>
+      field.dataType match {
+        case t: VectorUDT => StructField(field.name, VectorType)
+        case other => field
+      }
+    }.toArray)
+
+    spark.createDataFrame(rows, schema)
+  }
```

Comment:
Is this for backwards compatibility - does the schema of the `DataFrame`s use `mllib.Vector` and `VectorUDT`? Should we just run a job once to save them in the new format instead of adding in this code?

Reply:
Ideally, yes. Can we easily recreate the parquet files? Is there a script?

Reply:
Oh, I see what you mean. It should be easy enough to just load up the DataFrame, run this code, and then re-save it. I'll do that tomorrow morning.